rolling/distribution.yaml

```
@@ -3479,6 +3479,25 @@ repositories:
       url: https://github.com/osrf/osrf_testing_tools_cpp.git
       version: rolling
     status: maintained
+  ouster-ros:
+    doc:
+      type: git
+      url: https://github.com/ouster-lidar/ouster-ros.wiki.git
```

```suggestion
      url: https://github.com/ouster-lidar/ouster-ros.git
```

This is supposed to be the url of the repository on which to run the automatic code documentation job. Not a link to human readable documentation. That should go in the package.xml.